lib/commons/dom/is-focusable.js

```diff
@@ -70,3 +70,18 @@ dom.isNativelyFocusable = function(el) {
 	}
 	return false;
 };
+
+/**
+ * Determines if an element is in the focus order, but would not be if its
+ * tabindex were unspecified.
+ * @method insertedIntoFocusOrder
+ * @memberof axe.commons.dom
+ * @instance
+ * @param {HTMLElement} el The HTMLElement
+ * @return {Boolean} True if the element is in the focus order but wouldn't be
+ * if its tabindex were removed. Else, false.
+ */
+dom.insertedIntoFocusOrder = function(el) {
+	return (dom.isFocusable(el) && el.tabIndex > -1 &&
```

Comment:
Please put the `el.tabIndex > -1` in the first part of the expression. The matcher will run on every node on the page, so we should put the fastest part first.

Reply:
Done.